netflix-roulette/src/components/AppHome/components/GenresTabs/GenresTabs.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { Tabs, Tab } from "@material-ui/core";
+import { GENRES } from "../../../../utils";
+
+interface GenresTabsProps {
+  selectedGenre: string;
+  changeGenre(event: React.ChangeEvent<{}>, genre: string): void;
+}
+
+export const GenresTabs: React.FC<GenresTabsProps> = ({
```

Comment:
Using `React.FC` [may be considered a bad practice](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components).